package.json

```diff
@@ -24,8 +24,8 @@
     "publish:next": "yarn changeset publish --tag next",
     "version:studio": "yarn changeset version --snapshot studio-$(git rev-parse --short=7 HEAD) && yarn angular build",
     "publish:studio": "yarn changeset publish --tag studio",
-    "version:latest": "yarn changeset version && yarn angular build",
-    "publish:latest": "yarn changeset publish"
+    "version:v3": "yarn changeset version && yarn angular build",
+    "publish:v3": "yarn changeset publish --tag v3-latest"
```

Comment:
Set the dist-tag to `v3-latest`.

Wanted to do `v3` tag first, but NPM does not allow dist tags named that can be read as semver versions.

Reply:
hmm, that makes sense